backend/test/test_qa.py

```diff
@@ -138,7 +139,7 @@ def failed_qa_run_id(crawler_crawl_id, crawler_auth_headers, default_org_id):
     assert crawls[0]["id"] == crawler_crawl_id
     assert crawls[0]["activeQAState"]
     assert crawls[0]["activeQAStats"]
-    assert crawls[0]["lastQAState"]
```

Comment:
I think we can keep this in? Also should add another test with the right query params.